CelesteNet.Client/Components/CelesteNetChatComponent.cs

```diff
@@ -242,14 +348,29 @@ public CelesteNetChatComponent(CelesteNetClientContext context, Game game)
                 );
 
                 if (!Calc.BetweenInterval(_Time, 0.5f)) {
-                    offs += CelesteNetClientFont.Measure(text).X * scale;
-                    CelesteNetClientFont.Draw(
-                        "_",
-                        new(50f * scale + offs, UI_HEIGHT - 105f * scale),
-                        Vector2.Zero,
-                        fontScale,
-                        Color.White * 0.5f
-                    );
+
+                    if(CursorIndex == Typing.Length) {
```

Comment:
Please change the spacing here to match the rest of the project 😅 

```suggestion
                    if (CursorIndex == Typing.Length) {
```